lib/lab_manager/app/endpoints/computes.rb

```diff
@@ -132,11 +132,22 @@ class Compute < Base
 
         get '/:id/snapshots/:snapshot_id' do
           compute = ::Compute.find(params[:id])
-          action = compute.snapshots.find(params[:snapshot_id])
-          action.to_json
+          snapshot = compute.snapshots.find(params[:snapshot_id])
```

Comment:
Snapshot model needs to implement something like `reload_snapshot_data` and needs to be called unless ['false', 'f', '0'].include?(params[:cached])